web/main.js

```diff
@@ -553,10 +570,14 @@ function hashchange() {
 }
 function codechange() {
     if (phraseInput.value === "") {
-        dialButton.value = "CREATE WORMHOLE";
+        dialButton.value = "Get file";
+        dialButton.disabled = true;
+        dialButton.hidden = false;
     }
     else {
-        dialButton.value = "JOIN WORMHOLE";
+        dialButton.value = "Get file";
```

Comment:
We execute `dialButton.value = "Get file";` regardless the value of `phraseInput.value`. Also, "Get file" is the default value of the button, and it must not change at all, so we can remove these two lines

Reply:
You're right, I've removed those two lines.